src/manifolds/ProbabilitySimplex.jl

```diff
@@ -421,16 +452,119 @@ function riemannian_gradient!(M::ProbabilitySimplex, X, p, Y; kwargs...)
     return X
 end
 
+@doc raw"""
+    riemann_tensor(::ProbabilitySimplex, p, X, Y, Z)
+
+Compute the Riemann tensor ``R(X,Y)Z`` at point `p` on [`ProbabilitySimplex`](@ref) `M`.
+It is computed using [`RealProbabilityAmplitudes`](@ref) isometry with positive orthant of
+a sphere.
+"""
+riemann_tensor(::ProbabilitySimplex, p, X, Y, Z)
+
+function riemann_tensor!(::ProbabilitySimplex{N}, Xresult, p, X, Y, Z) where {N}
+    ME = RealProbabilityAmplitudes(N)
+    pe = embed(ME, p)
+    Xrs = riemann_tensor(Sphere(N), pe, embed(ME, p, X), embed(ME, p, Y), embed(ME, p, Z))
+    project!(ME, Xresult, pe, Xrs)
+    return Xresult
+end
+
 function Base.show(io::IO, ::ProbabilitySimplex{n,boundary}) where {n,boundary}
-    return print(io, "ProbabilitySimplex($(n); boundary=$boundary)")
+    return print(io, "ProbabilitySimplex($(n); boundary=:$boundary)")
 end
 
 @doc raw"""
     zero_vector(M::ProbabilitySimplex, p)
 
-returns the zero tangent vector in the tangent space of the point `p`  from the
+Return the zero tangent vector in the tangent space of the point `p`  from the
 [`ProbabilitySimplex`](@ref) `M`, i.e. its representation by the zero vector in the embedding.
 """
 zero_vector(::ProbabilitySimplex, ::Any)
 
 zero_vector!(::ProbabilitySimplex, X, p) = fill!(X, 0)
+
+@doc raw"""
+    RealProbabilityAmplitudes
+
+An explicit isometric embedding of interior of [`ProbabilitySimplex`] in
+positive orthant of the [`Sphere`]. Some properties extend to the boundary but not all.
+
+This embedding isometrically maps the Fisher-Rao metric on the open probability simplex to
+the sphere of radius 1 with Euclidean metric. More details can be found in Section 2.2
+of [^AyJostLeSchwachhöfer2017].
+
+The name derives from the notion of probability amplitudes in quantum mechanics.
+They are complex-valued and their squared norm corresponds to probability. This construction
+restricted to real valued amplitudes results in this embedding.
+
+# Constructor
+
+    RealProbabilityAmplitudes(n)
+"""
+const RealProbabilityAmplitudes{N} = EmbeddedManifold{ℝ,<:ProbabilitySimplex{N},Sphere{N,ℝ}}
+
+function RealProbabilityAmplitudes(n)
+    return EmbeddedManifold(ProbabilitySimplex(n), Sphere(n))
+end
+
+@doc raw"""
+    embed(M::RealProbabilityAmplitudes{N}, p) where {N}
+
+Convert point `p` on `ProbabilitySimplex` to (real) probability amplitude. The formula reads
+``(\sqrt{p_1}, \sqrt{p_2}, …, \sqrt{p_{N+1}})``. This is an isometry from the interior of
+the probability simplex to the interior of the positive orthant of a sphere.
+"""
+function embed(M::RealProbabilityAmplitudes, p)
+    return embed!(M, similar(p), p)
```

Comment:
Why does this not work on the default with allocate and such?
Or phrased differently: I expected ember only to be here for the doc-string not for an actual implementation.